migrate.go

```diff
@@ -202,7 +201,7 @@ func EnsureDBVersion(db *sql.DB) (int64, error) {
 		toSkip = append(toSkip, row.VersionId)
 	}
 
-	panic("unreachable")
```

Comment:
This looks like a distributed race now. I think we should use a distributed lock (think etcd) here to ensure that migrations are run only once. Consider what happens when two nodes start migrating data in parallel.